src/util/OAuth2Util.js

```diff
@@ -95,6 +95,43 @@ define(['okta', './Enums', './Errors'], function (Okta, Enums, Errors) {
     }
   };
 
+  // Parse through the OAuth 'authParams' object to ensure the 'openid' scope is
+  // included (if required)
+  util.scrubScopes = function (authParams) {
```

Comment:
The name of this method is too vague.  Similar to my comments about `filterOAuthRedirectParams`, I do wonder if this really needs to be it's own method.